examples/native/components/ControlExamples/Number/Number.stories.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { ComponentStory, ComponentMeta } from '@storybook/react-native';
+import { Multiply } from './Number';
+
+const NumberMeta: ComponentMeta<typeof Multiply> = {
+  title: 'Number control',
+  component: Multiply,
+};
+
+export default NumberMeta;
+
+type NumberStory = ComponentStory<typeof Multiply>;
+
+export const Basic: NumberStory = (args) => <Multiply {...args} />;
```

Comment:
This behaves in a strange way for me on the Android Emulator, can you reproduce it? Seems to work fine on the iOS emulator, can't reproduce the issue there.

1. Open the story Navigator.
2. Select the "Basic" story.
3. Open "Addons -> Controls"
4. Click on the "first" field so that the keyboard comes up.
5. Click back to "Navigator"

## What happens
The story list in the navigator gets truncated a lot (seems to only occupy ~50% or so of the screen) after Step 5.

## Expected result
The story list would be rendered in the same way in Step 5 as it was in Step 1.

### Screenshot after Step 1
![Screen Shot 2021-07-11 at 20 08 59](https://user-images.githubusercontent.com/6605505/125204064-01482d80-e284-11eb-92df-5fd128d83757.png)

### Screenshot after Step 4
![Screen Shot 2021-07-11 at 20 11 45](https://user-images.githubusercontent.com/6605505/125204150-73b90d80-e284-11eb-8df0-ca9f0d8b76d2.png)

### Screenshot after Step 5
![Screen Shot 2021-07-11 at 20 13 15](https://user-images.githubusercontent.com/6605505/125204157-7ae01b80-e284-11eb-81c1-ba87706195e1.png)

Reply:
That's weird, I think it's related to the keyboard aware component. I'm not able to check now but I'll have a look tomorrow.

Reply:
Will create an issue to tackle this bug, seems to be for all text inputs on knobs/controls